SS_objfunc.tpl

```diff
@@ -321,9 +321,29 @@ FUNCTION void evaluate_the_objective_function()
         }
         length_like(f,i)-=temp;
       }
-      else   //  MV_Tweedie
+      else  //  multivariate-Tweedie
```

Comment:
I think this we should just assign 3 to this new option and changed this line to 
`if( (Comp_Err_L(f)==3)`
otherwise I think multinomial likelihood will trigger both the section above with
`if( (Comp_Err_L(f)==0)`
and also this new section

Reply:
makes sense!  Thanks for catching that.  I can't compile and run it without parameter declarations, so bound to be a few issues like this.  

did you make the change in the PR code?  I'm not used to code reviews during PR, but seems great :)

Reply:
No, just added a comment on the code change (by clicking on line under "Files changed" under pull request on github.com, but I haven't made any change yet. I think that I could either push a change to the new branch on your fork to be included in this PR, or we could merge your PR into the branch on the nmfs-stock-synthesis version and then I can push the change to the branch there. Since nobody is using the branch for anything but this likelihood, it makes sense to merge first.